tomls/permit-all-transferCrossChain.toml

```diff
@@ -1,3 +1,14 @@
+[setting.ccip_router]
```

Comment:
for simplicity should we leave ccip disabled for now?

Reply:
this one is not for andromeda. This is a generic config and this bit was moved exactly to avoid it being used in andromeda deployment

Reply:
@noahlitvin

Reply:
It was actually Noah who made this change hehe

Reply:
> It was actually Noah who made this change hehe

ah makes sense thanks